assets/css/payment_template.css

```diff
@@ -0,0 +1,297 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+
+body,
+html {
+	background: transparent linear-gradient(254deg, #d500f9 0, #2962ff 99%) 0 0 no-repeat padding-box;
+	font-family: "Open Sans", sans-serif;
+	min-height: 100%;
+	overflow-x: hidden;
+	width: 100%;
+}
+
+.container {
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	margin: auto;
+	max-width: 1200px;
+	padding: 0 5px;
+	width: 100%;
+}
+
+.title {
+	color: #fff;
+	font-family: "Days One", sans-serif;
+	font-size: 23px;
+	font-weight: 600;
+	line-height: 32px;
+	padding: 30px 0;
+	text-align: center;
+}
+
+.row {
+	align-items: stretch;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	margin-bottom: 30px;
+	width: 100%;
+}
+
+.column {
+	flex: 1 1 0;
+	margin: 7px 16px;
+}
+
+.one-quarter {
+	flex: none;
+	width: calc(100% * 1/4);
+}
+
+.paragraph-center {
+	color: #fff;
+	text-align: center;
+}
+
+.card {
+	background-color: #fff;
+	border-radius: 20px;
+	box-shadow: 0 6px 24px rgba(0, 0, 0, 0.29);
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+	padding: 20px;
+	position: relative;
+	transition: transform 0.2s, box-shadow 0.2s;
+	width: 100%;
+}
+
+.card:hover {
+	box-shadow: 0 8px 26px rgba(0, 0, 0, 0.25);
+	transform: scale(1.15);
+	z-index: 100;
+}
+
+.zoom {
+	box-shadow: 0 8px 26px rgba(0, 0, 0, 0.25);
+	transform: scale(1.1);
+	z-index: 99;
+}
+
+
+.card__title {
```

Comment:
Codacy found an issue: [Selector should use lowercase and separate words with hyphens (selector-class-pattern)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5907666)